index.html

```diff
@@ -54,7 +56,7 @@
       j.async = true;
       j.src = '//www.googletagmanager.com/gtm.js?id=' + i + dl;
       f.parentNode.insertBefore(j, f);
-    })(window, document, 'script', 'dataLayer', 'GTM-K3K9K9');
+    })(window, document, 'script', 'dataLayer', 'GTM-K3K9K9', navigator.doNotTrack);
```

Comment:
What does this addition do? Why not just enclose the whole function in a single `if (navigator.doNotTrack !== "1")` block?